portal/templates/portal.html

```diff
@@ -18,55 +17,63 @@ <h2 class="tnth-subhead">{{ _("Welcome to TrueNTH") }}</h2>
 </div>
 {% endif %}
 
+
 {# present each intervention in turn if configured for user #}
 {% for intervention in interventions %}
     {% set display = intervention.display_for_user(user) %}
     {% if display.access %}
         <div class="portal-item">
             {% if intervention.description and intervention.description != "" %}<h2 class="tnth-subhead">{{ intervention.description }}</h2>{% endif %}
-            <div>
-                {{ display.card_html | safe }}
-            </div>
-            <div>
-                {% if display.link_url is none %}
-                <div>
-                    <br/>
-                    <a class="btn-lg btn-tnth-primary disabled" href="#">
-                        {{ display.link_label }}
-                    </a>
-                </div>
-                {% else %}
-                <div>
-                    <br/>
-                    <a class="btn-lg btn-tnth-primary" href="{{ display.link_url }}">
-                        {{ display.link_label }}
-                    </a>
+            {% if display.card_html %}
+                <div class="portal-description">
+                    {{ display.card_html | safe }}
                 </div>
-                {% endif %}
-            </div>
+            {% endif %}
+            
+            {% if display.link_url is none %}
```

Comment:
I think the following check might be more appropriate, in case `link_url` is an empty string and not just None/null. I may be wrong though...

```
{% if not display.link_url %}
```